packages/manager/apps/hub/webpack.config.js

```diff
@@ -49,6 +49,12 @@ module.exports = (env = {}) => {
         __NG_APP_INJECTIONS__: process.env.NG_APP_INJECTIONS
           ? `'${process.env.NG_APP_INJECTIONS}'`
           : 'null',
+        __FEEDBACK_URL_EN__: process.env.FEEDBACK_URL_EN
```

Comment:
I don't get what you are doing, the URL is injected at build time ? Why not hardcoding or APIing it ?

Reply:
@FredericEspiau we talked about it with @antleblanc this will be changed